doc/classes/ProjectSettings.xml

```diff
@@ -652,13 +655,6 @@
 			See [enum DisplayServer.VSyncMode] for possible values and how they affect the behavior of your application.
 			Depending on the platform and used renderer, the engine will fall back to [code]Enabled[/code], if the desired mode is not supported.
 		</member>
-		<member name="dotnet/project/assembly_name" type="String" setter="" getter="" default="&quot;&quot;">
-			Name of the .NET assembly. This name is used as the name of the [code].csproj[/code] and [code].sln[/code] files. By default, it's set to the name of the project ([member application/config/name]) allowing to change it in the future without affecting the .NET assembly.
-		</member>
-		<member name="dotnet/project/solution_directory" type="String" setter="" getter="" default="&quot;&quot;">
-			Directory that contains the [code].sln[/code] file. By default, the [code].sln[/code] files is in the root of the project directory, next to the [code]project.godot[/code] and [code].csproj[/code] files.
-			Changing this value allows setting up a multi-project scenario where there are multiple [code].csproj[/code]. Keep in mind that the Godot project is considered one of the C# projects in the workspace and it's root directory should contain the [code]project.godot[/code] and [code].csproj[/code] next to each other.
-		</member>
```

Comment:
I just ran `doctool` and didn't look at the result. I'll fix it.

Reply:
For the record, if you rebase on latest master these settings will be kept (but you still need to re-add them manually to keep the descriptions).

Reply:
I rebased and restored the file with `git restore -s master doc/classes/ProjectSettings.xml` then rebuilt and ran `doctool` again. This does discard the description I added but since it's on the commit I can simply discard that hunk (using `git restore -p doc/classes/ProjectSettings.xml`). Just sharing some git-fu here on how to avoid having to copy-paste the descriptions back.